mxd/1_MXD_Introduction.md

```diff
@@ -0,0 +1,50 @@
+# 1 Introduction
+
+With the Minimum Tractus-X Dataspace, IT departments can set up their own little data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.
+
+For whom is that relevant?
+This tutorial is designed for companies that want to perform data exchange in a "real" Catena-X data space infrastructure.
+
+Who should execute the tutorial?
```

Comment:
move to intended audience. And you don't "execute" a tutorial. You "read" it, or fwiw you "do" it.